python/foglamp/services/south/ingest.py

```diff
@@ -126,12 +119,6 @@ class Ingest(object):
         category = "{}Advanced".format(cls._parent_service._name)
 
         default_config = {
-            "write_statistics_frequency_seconds": {
```

Comment:
How is this actually controlled? Maybe the problem is not that we should remove the unused config, but rather we soul be making use of it.

Reply:
This is removed via PR https://github.com/foglamp/FogLAMP/pull/1094/files 

More detailed info is in PR comments e.g. https://github.com/foglamp/FogLAMP/pull/1173#issuecomment-428528544 /cc @amarendra-dianomic

Reply:
@MarkRiddoch write_statistics_frequency_seconds was in use earlier when we used to execute _write_statistics() via a task scheduled to run every **write_statistics_frequency_seconds** seconds, independent of _insert_readings() task. Now in FOGL-1824, this behaviour has been changed and now _write_statistics() is being run at the end of processing of each readings queue in _insert_readings() task itself. Since _write_statistics() is no longer being run independently,  write_statistics_frequency_seconds has become redundant and hence removed in this PR.